Gemfile

```diff
@@ -16,6 +16,8 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rubocop', require: false
   gem 'shoulda-matchers', '~> 3.1'
+  gem "capybara"
+  gem "selenium-webdriver"
```

Comment:
Do we need Selenium (or similar) yet?

Reply:
Not yet, removed selenium for now. Thanks!